designer/client/src/LinkCreate.test.tsx

```diff
@@ -12,16 +13,17 @@ import React from 'react'
 import { LinkCreate } from '~/src/LinkCreate.jsx'
 import { RenderWithContext } from '~/test/helpers/renderers.jsx'
 
-const data: FormDefinition = {
+const data = {
   pages: [
     {
       title: 'First page',
       path: '/first-page',
       components: [
         {
-          type: ComponentType.YesNoField,
           name: 'ukPassport',
           title: 'Do you have a UK passport?',
+          type: ComponentType.YesNoField,
+          subType: ComponentSubType.Field,
```

Comment:
I've updated all types and test fixtures to require `type` and `subType`

Previously saved JSON won't necessarily have these so `subType` is still optional via Joi